.github/scripts/build.sh

```diff
@@ -2,6 +2,17 @@
 
 source `dirname ${BASH_SOURCE[0]}`/config.sh
 
+if [ "$CCACHE" = 1 ] ; then
+  pushd /usr/lib/ccache
+    if ! [ -f $TARGET-gcc ]; then
+      sudo ln -s ../../bin/ccache $TARGET-gcc
```

Comment:
Is there any other way to avoid sudo call?

Reply:
Yes, create `aarch64-w64-mingw32` symlinks in custom directory and add that to the `PATH`.

Reply:
Yes, that's what I had in mind (if you mean `PATH=/usr/lib/ccache/:$TOOLCHAIN_PATH/ccache:$TOOLCHAIN_PATH/bin:$PATH`).

Reply:
looks good to me